src/RapidezStatamicServiceProvider.php

```diff
@@ -54,6 +59,20 @@ public function bootCommands() : self
         return $this;
     }
 
+    private function getGlobals() : array
+    {
+        return Cache::rememberForever('statamic-globals', function() {
```

Comment:
Maybe add the locale to the cache key here, so every locale will get their own cache. Otherwise if you have a EN and ES store they both get the same.